File: sdk/python/src/example_constants.py
from __future__ import print_function
import time
import io_stockx
from io_stockx.rest import ApiException
from pprint import pprint

class ExampleConstants:

    AWS_API_KEY = "<API Key>"

    STOCKX_USERNAME = "<StockX Username>"

    STOCKX_PASSWORD = "<StockX Password>"

    DEMO_PRODUCT_ID = "air-jordan-1-retro-high-off-white-chicago"

    DEMO_CUSTOMER_ID = "1471698"

    ENABLE_DEBUG = True

    JWT_HEADER = "Jwt-Authorization"

File: sdk/csharp/src/IO.StockX.Examples/ExampleConstants.cs

namespace Example
{
    static public class ExampleConstants
    {
        /** The AWS API key provided by StockX to access StockX APIs. */
        public const string AWS_API_KEY = "<Your AWS API Key>";

        /** The StockX base url to call */
        public const string STOCKX_ENDPOINT = "https://gateway.stockx.com/stage";

         /** The StockX account password to authenticate with. */
        public const string STOCKX_USERNAME = "<StockX account username>";

        /** The StockX account password to authenticate with. */
        public const string STOCKX_PASSWORD = "<StockX account password>";

        /** The product ID to lookup */
        public const string DEMO_PRODUCT_ID = "air-jordan-1-retro-high-off-white-chicago";

        /** The custoner ID to lookup orders for. */
        public const string DEMO_CUSTOMER_ID = "1471698";

        /** JWT bearer token header name. */
        public const string JWT_HEADER = "Jwt-Authorization";

    }

}